src/pages/_app.tsx

```diff
@@ -32,7 +32,8 @@ const MyApp = ({ Component, pageProps }: AppProps) => {
     sal();
   }, []);
   useEffect(() => {
-    document.body.className = `${pageProps.className}`;
+    /* document.body.className = `${pageProps.className}`; */
```

Comment:
Remove this commented code, I think we don't need it anymore (template string value)